citest/gcp_testing/google_session.py

```diff
@@ -73,7 +74,8 @@ def __init__(
   def logout(self):
     """Log out of the current authenticated Google session.
     """
-    raise NotImplementedError()
+    self.__session.get("https://www.google.com/accounts/Logout")
+    self.__session.cookies.clear()
```

Comment:
This makes future calls the the scope of this session anonymous/unauthenticated, since `cookies` would be empty. This would cause a redirect if you make a subsequent call to try to re-access the same service, since it no longer recognizes your session.